data/core/units/ogres/Great_Ogre.cfg

```diff
@@ -0,0 +1,86 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Great Ogre
+    name= _ "Great Ogre"
+    race=ogre
+    image="units/ogres/great-ogre.png"
+    hitpoints=76
+    movement_type=largefoot
+    [resistance]
+        arcane=80
+    [/resistance]
+    movement=5
+    profile="portraits/monsters/ogre.webp"
+    experience=150
+    level=3
+    alignment=neutral
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=43
+    usage=fighter
+    description= _ "Great Ogres are giant creatures that usually live alone in the wilderness, remarkably similar to humans in form, though large and misshapen. While they can be easily outrun or outsmarted, their strength is not to be underestimated."
+    die_sound={SOUND_LIST:OGRE_DIE}
+    [attack]
+        name=cleaver
+        #textdomain wesnoth-units
```

Comment:
the ephemeral textdomain change here is not required now since this is entirely in `#textdomain wesnoth-units`